06_discord_bot/06_discord_bot.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Créer son premier Bot pour discord en python
+date: 2020
+author:
+- Gabrielle Pauvert
+---
+
+Introduction
+============
+
+Dans ce TD, nous allons apprendre pas à pas comment créer un bot pour Discord en python et lui donner quelques premières commandes très simples, par exemple répondre à une commande par un “hello world”.
+
+Création du Bot
+===============
+
+Tout d’abord, connectez vous sur le site de gestion des bots discord :
+<https://discordapp.com/developers/docs/intro>.
+
+Puis cliquez sur l’onglet `Applications` en haut à gauche.
+
+[comment]: ![image](ims/im1.png)
+<center>
+<img src="ims/im1.png" alt="centered image" width="500"/>
+</center>
```

Comment:
The LaTeX produced by pandoc doesn't seem to include these images :(

Here's how we do it in other subjects: https://github.com/prologin/gcc-resources/blame/master/05_projet_microbit/05_projet_microbit.md#L19